### PR TITLE
Docker support

### PR DESCRIPTION
**Pull request by MyGodItsFull0fStars**
Added a Dockerfile to execute VCA from a Docker container.

This is done to ensure that systems that otherwise can't execute VCA are able to do so via Docker.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 build
-.vscode
\ No newline at end of file
+.vscode
+docs/build.md
```
Comment (ChristianFeldmann) [inline .gitignore]: Why would you gitignoe that documentation flle?

**Review by ChristianFeldmann — CHANGES_REQUESTED** [review]
Just one tiny thing. Otherwise nice. Even more ways to use VCA on platforms where it may not compile natively.

File: .gitignore (hunk 1 above)
Comment (MyGodItsFull0fStars) [reply .gitignore]: I seem to have added it by accident. Thank you for pointing that out. Will change that. :)

**Comment by ChristianFeldmann** [comment]
I think we have to update the runners first (https://github.com/cd-athena/VCA/pull/81) before we can merge this.

**Comment by MyGodItsFull0fStars** [comment]
Apparently, these two OS variants failed for quite some time. At least I saw that the same variants failed on the `stable` branch.

**Review by ChristianFeldmann — APPROVED** [review]

**Pull request merged**